config/widgets/settings/adventureOptionsTab.json

```diff
@@ -31,31 +31,31 @@
 			"items":
 			[
 				{
-					"index": 1,
+					"index": 200,
 					"type": "toggleButton",
 					"image": "sysopb1",
 					"help": "core.help.349",
 					"position": {"x": 0, "y": 0}
 				},
 
 				{
-					"index": 2,
+					"index": 150,
 					"type": "toggleButton",
 					"image": "sysopb2",
 					"help": "core.help.350",
 					"position": {"x": 48, "y": 0}
 				},
 
 				{
-					"index": 4,
+					"index": 100,
 					"type": "toggleButton",
 					"image": "sysopb3",
 					"help": "core.help.351",
 					"position": {"x": 96, "y": 0}
 				},
 
 				{
-					"index": 16,
+					"index": 0,
```

Comment:
I did not manually test this, but note that in H3 thi speed is not exactly instant - for example try to move from one corner of XL map to another

Reply:
Yes, but it seems to be extremely fast, like 1 tile per ms. Too fast for my video capture measurements.

At this point I would call it instant and any visual delays were likely not intended.
It is not instant in vcmi either, mostly due to server-client communication delays